charlesinto/iReporter
# feat(endpoint): Get an Intervention Record

#### What does this PR do?

- Get intervention record

#### Description of Task to be completed?
- Implement jwt authentication
- Verify user token
- User should be to get an intervention record

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start  the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `GET` request to `http://localhost:7000/api/v1/interventions/:id`
- You should receive a successful response

#### Any background context you want to provide?


`

#### What are the relevant pivotal tracker stories?
[#162588758](https://www.pivotaltracker.com/story/show/162588758)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/26615085/49825858-ba20f580-fd85-11e8-819c-043a570a2e07.png)





#### Questions:
N/A
